cmd/probe/eventhandler.go

```diff
@@ -22,11 +22,14 @@ import (
 	libudevwrapper "github.com/openebs/node-disk-manager/pkg/udev"
 )
 
+// EventAction global variable to store eventAction
```

Comment:
`EventAction` is a type not variable please update.

Reply:
okay, changing variable to type